.azure-pipelines/stage-aarch64.yml

```diff
@@ -0,0 +1,45 @@
+stages:
```

Comment:
this should go to https://github.com/aio-libs/azure-pipelines

Reply:
As mentioned above stage-aarch64.yml(renamed as stage-test-aarch64) is for running pytest for aarch64, same as stage-test.yml is for x86, Mac and Windows.
In my opinion it should be kept in same path as stage-test.yml. Please give your feedback and suggestions regarging this.